src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingContext.cs

```diff
@@ -194,25 +196,37 @@ public async Task<FormattingContext> WithTextAsync(SourceText changedText)
             var engine = OriginalSnapshot.Project.GetProjectEngine();
             var importSources = new List<RazorSourceDocument>();
 
-            if (OriginalSnapshot is DefaultDocumentSnapshot documentSnapshot)
+            var imports = OriginalSnapshot.GetImports();
+            foreach (var import in imports)
             {
-                var imports = documentSnapshot.State.GetImports((DefaultProjectSnapshot)OriginalSnapshot.Project);
```

Comment:
The `GetImports()` method on `DefaultDocumentSnapshot` does the same thing as this line did anyway, and this is easier to mock.
https://github.com/dotnet/aspnetcore-tooling/blob/main/src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultDocumentSnapshot.cs#L46